code/ship/ship.cpp

```diff
@@ -14276,7 +14276,7 @@ int ship_do_rearm_frame( object *objp, float frametime )
 				{
 					// Goober5000
 					gamesnd_id sound_index;
-					if (gamesnd_game_sound_valid(GameSounds::BALLISTIC_START_LOAD))
+					if (gamesnd_game_sound_exists(GameSounds::BALLISTIC_START_LOAD))
```

Comment:
Is the change here and similar changes below a concern (e.g., potential perf issue) in that `gamesnd_game_sound_valid()` runs fast and has no side effects, whereas the new function doesn't have those guarantees?

I assume it's not a problem but figured I should ask.

Reply:
It would not be a problem in this case because if the sound can be loaded, it will be used.